README.md

```diff
@@ -37,6 +37,8 @@ In the project properties for **pdf-java-toolkit-samples**, select the **Maven**
 
 It will ask if you want to update the product configuration. Click **Yes**.
 
+The samples were developed and tested using [Mars](https://eclipse.org/mars/)
```

Comment:
It looks like we have a few different versions in use, so does it make sense in the end to just say "Mars"? @datalogics-bhaugen @datalogics-kam